snuba/datasets/events_processor.py

```diff
@@ -1,8 +1,7 @@
+import logging
```

Comment:
it seems correct to me.  What's the issue ?

Reply:
Not sure in retrospect. I may have been looking at an outdated version or just misplaced this comment entirely. This is inconsistent with when I apply it locally, but that seems more of a matter of `_strptime` being incorrectly categorized as an application package (I don't think that was what I was thinking about before but honestly don't remember):

```diff
diff --git a/snuba/datasets/events_processor.py b/snuba/datasets/events_processor.py
index e74995ec..8e908e18 100644
--- a/snuba/datasets/events_processor.py
+++ b/snuba/datasets/events_processor.py
@@ -1,7 +1,7 @@
-import _strptime  # NOQA fixes _strptime deferred import issue
 import logging
 from typing import Any, Mapping, MutableMapping
 
+import _strptime  # NOQA fixes _strptime deferred import issue
 from snuba.clickhouse.columns import ColumnSet
 from snuba.consumer import KafkaMessageMetadata
 from snuba.datasets.events_format import extract_http, extract_user
```

Aside, I don't really recall what this was for in the first place but it seems like if this is still necessary ([seems like it may have been fixed in Python 3?](https://stackoverflow.com/questions/32245560/module-object-has-no-attribute-strptime-with-several-threads-python)) this should probably be moved to the top level `__init__` and done once rather than being strewn all over: https://github.com/getsentry/snuba/search?q=_strptime